.github/workflows/topos:integration-tests.yml

```diff
@@ -67,10 +68,17 @@ jobs:
           cd infra &&
           npm install
 
+      - name: Setup environment
+        run: |
+          cd infra
+          cat .env | grep -v '^\s*$\|^\s*\#' | xargs -I {} echo "{}" >> $GITHUB_ENV
+          echo "LOCAL_ERC20_HOME=${PWD}" >> $GITHUB_ENV
+          echo "TOPOS_MESSAGING_PROTOCOL_CONTRACTS_VERSION=${{ env.CONTRACTS_DOCKER_TAG }}" >> $GITHUB_ENV
+          echo "TOPOS_VERSION=${{ env.TOPOS_DOCKER_TAG }}" >> $GITHUB_ENV
+
       - name: Start local ERC20 messaging infra containers
         run: >
           cd infra &&
-          source .env &&
```

Comment:
This was a bug in the test indeed overwriting reference set when the test is trigered